schemas/headers.yaml

```diff
@@ -4,7 +4,7 @@ info:
   title: Headers Sample
   description: A sample API that demonstrates header parameters/responses
 paths:
-  /info:
+  /headers/info:
```

Comment:
Because the `TestApp` needs all these in one project, I needed to add prefixes. Feature possibility: add prefix via options. 🤔